functions/src/telegram/utils.ts

```diff
@@ -1,4 +1,25 @@
+import { fetch } from 'undici'
+
 export const addUsernameCommand = (command: string) => [
   command,
   `${command}@${process.env.TELEGRAM_BOT_USERNAME}`,
 ];
+
+export const getRequest = async (url: string) => {
+  const res = await fetch(url)
+  const json = await res.json() 
+
+  return JSON.stringify(json);
+};
+
+export const getListGithub = async (url: string) => {
```

Comment:
```suggestion
export const getListOfPrOrIssue = async (url: string) => {
```

se vi vengono in mente nomi migliori per le funzioni proponete pure

Reply:
Creerei una funzione singola che si chiama `getDataFromGithub`. Questa funzione poi viene chiamata da

- `getIssues` (la quale aggiunge all'url, la parte delle issue)
- `getPRs` (la quale aggiunge all'url, la parte delle PR)

`getDataFromGithub` quindi tiene come punto comune l'URL e le operazioni di processing.
Così non devi passare ovunque l'URL intero.

Reply:
`getIssues` e `getPRs` però sarebbero completamente uguali, l'unica differenza è che a `getDataFromGithub` viene passato come parametro `issues`/`pulls`

Reply:
Non ha importanza. Sono unità diverse.

```js
async function getIssue() {
    const response = await getDataFromGithub("/issues?state=open");
    return parseGithubResponse(response);
}

async function getPullRequests() {
    const response = await getDataFromGithub("/pulls?state=open");
    return parseGithubResponse(response);
}
```

Reply:
Va bene, ora modifico. La funziona che c'era già su `utils.ts` era una arrow function, le faccio tutte così oppure no?

Reply:
Personalmente preferisco le `function`. Vedi un po' come è il resto del codice e mantieni uno stile uniforme

Reply:
I file dentro `/commands` sono tutte `function`, mentre `setLastMemberActivity.ts` e `addUsernameCommand()` (è su utils) sono arrow function

Reply:
Allora fai come vuoi, prima o poi uniformeremo tutto 😂